sdk/flutter/lib/picovoice.dart

```diff
@@ -119,25 +154,24 @@ class Picovoice {
   /// [frame] A frame of audio samples. The number of samples per frame can be attained by calling
   /// `.frameLength`. The incoming audio needs to have a sample rate equal to `.sample_rate` and be 16-bit linearly-encoded.
   /// Picovoice operates on single-channel audio.
-  void process(List<int> frame) {
+  void process(List<int> frame) async {
```

Comment:
can we add a check for null frame and frame length here?

Reply:
only check for frame length since null can't be passed it